### PR TITLE
[Tests] Refactor MockDownloadManager

### PR DESCRIPTION
**Pull request by patrickvonplaten**
Clean mock download manager class. 
The print function was not of much help I think. 
We should think about adding a command that creates the dummy folder structure for the user.

### CODE REVIEW

File: datasets/wiki40b/wiki40b.py [hunk 1]
```diff
@@ -20,9 +20,9 @@
 import logging
 import os
```
Comment (patrickvonplaten) [inline datasets/wiki40b/wiki40b.py:22]: Now this looks like we have different versions of isort :-/

Comment (lhoestq) [reply datasets/wiki40b/wiki40b.py:22]: I missed this bug so much

File: 1 [hunk 2]
```diff
@@ -0,0 +1,157 @@
+import os
```
Comment (lhoestq) [inline 1:1]: I'm not sure this file is that the right place nor have the right name ^^ Looks like it's called `1` and is placed at the root

Comment (patrickvonplaten) [reply 1:1]: uuups :D thanks :-)

**Pull request merged**